### ISSUE TITLE
Issues

### ISSUE THREAD

**Issue by jiggy04** [jiggy04/Hello-world#1]
I am just starting as a beginner with github, is there any benefit of using and studying Git?

**Comment by Allan2000-Git** [comment]
yes a lot. Github is like your technical resume. Always upload your work on github. It might a small project or a big. Maintain your github profile by adding your details on your readme.md file. 

https://www.freecodecamp.org/news/how-to-write-a-good-readme-file/

**Comment by Damsith-LK** [comment]
@Allan2000-Git Fixes issue #1 :)

**Comment by Dudu-Passoni** [comment]
Git is a distributed version control system that tracks changes in any set of computer files, usually used for coordinating work among programmers who are collaboratively developing source code during software development. Its very important to know Git to become a well-rounded developer.

**Comment by jiggy04** [comment]
Good Morning. Thanks for the words of encouragement. Actually, I am new to the world of Git and Tech in general. I was practicing using Microsoft sandbox. I made a couple of errors and I couldn't correct them. Hopefully with more practice, I would be able to understand the steps properly and how to correct mistakes. thanks again.
________________________________
